habitat/plan.sh

```diff
@@ -1,68 +1,85 @@
 pkg_name=inspec
 pkg_origin=chef
-pkg_version=$(cat "VERSION")
 pkg_description="InSpec is an open-source testing framework for infrastructure
   with a human- and machine-readable language for specifying compliance,
   security and policy requirements."
-pkg_upstream_url=https://github.com/chef/inspec
+pkg_upstream_url=https://www.inspec.io/
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
-pkg_source=false
 pkg_deps=(
+  core/busybox-static
+  core/cacerts
   core/coreutils
   core/libxml2
   core/libxslt
-  core/ruby
   core/net-tools
-  core/busybox-static
+  core/ruby
 )
 pkg_build_deps=(
   core/bundler
-  core/coreutils
   core/gcc
   core/make
   core/readline
+  core/sed
 )
 pkg_bin_dirs=(bin)
 
-do_download() {
-  return 0
-}
-
-do_verify() {
-  return 0
+pkg_version() {
+  cat VERSION
 }
 
-do_unpack() {
-  return 0
+do_before() {
+  if [[ ! -f VERSION ]]; then
+    exit_with "habitat/VERSION not found. Run 'rake release_habitat' to ensure it exists" 1
+  fi
+  update_pkg_version
 }
 
 do_prepare() {
-  # Create a Gemfile with what we need
-  cat > Gemfile <<GEMFILE
+  # If we use the Gemfile in the project ($SRC_PATH/Gemfile), and run `bundle
```

Comment:
I think this is avoidable: https://github.com/habitat-sh/core-plans/blob/master/scaffolding-ruby/lib/scaffolding.sh#L162-L192 but it's fine for this. It'd be cool to get the ruby scaffolding working for ruby bins